paper/basic_training.tex

```diff
@@ -805,6 +805,89 @@ \subsection{Long range electrostatics}
 \item How to choose parameters
 \end{itemize}
 
+
+\subsubsection{Motivation}
+
+Calculation of non-bonded interaction is generally the most time-consuming step of classical energy calculation. While the elements(?) for bonded interaction remain unchanged during the course of MD simulation, the ones for non-bonded interactions have to be updated frequently. Furthermore, $r^{-1} $ dependence  of coulombic interaction in a 3D space complicates the calculation in the following two ways :
+\begin{itemize}
+\item Coulombic potential is conditionally convergent. What do we mean by this? Consider for example the alternating harmonic series : \\
+\[
+\sum_{k=1}^{\infty} \frac{-1^{k+1}}{k} = 1 - \frac{1}{2} + \frac{1}{3} - \frac{1}{4} + \frac{1}{5} - ... = ln 2 
+\]
+
+But if we now change the order of the summation to, lets say :
+\[
+( 1 - \frac{1}{2})  - \frac{1}{4} + (\frac{1}{3} - \frac{1}{6}) - \frac{1}{8} + (\frac{1}{5} - \frac{1}{10}) - \frac{1}{12} + (\frac{1}{7} - \frac{1}{14}) - \frac{1}{16} + .. 
+\]
+\[
+= \frac{1}{2} - \frac{1}{4} + \frac{1}{6} - \frac{1}{8} + \frac{1}{10} - ... 
+\]
+\[
+= \frac{1}{2} (1 - \frac{1}{2} + \frac{1}{3} - \frac{1}{4} + \frac{1}{5} - ...) = \frac{1}{2} ln 2
+\]
+
+Using a different order might give us a different result. Thus the order of summation matters !
+
+\item Coulombic potential is a slow-decaying, long range potential. This means that we need to account for non-bonded pairs over larger pairwise distances in order to minimize the errors. As enlisting atom-pairs is fundamentally an $O(n^2) $ operation, the time taken to compute non-bonded pairs increases quadratically with respect to an increase in the number of atoms.   
```

Comment:
If you don't mind, please lay out your TeX in a one-sentence-per-line style, which plays better with Git's version control. Right now if I, say, make an edit that touches one word in one of your paragraphs the diff will make it look like the whole paragraph was edited, which makes it hard to see what was changed.